test/lib_i2s/backpressure_test/src/main.c

```diff
@@ -79,7 +85,11 @@ i2s_restart_t i2s_restart_check(void *app_data)
     return I2S_NO_RESTART;
 }
 
+#if DATA_BITS == 32
 #define OVERHEAD_TICKS 185 // Some of the period needs to be allowed for the callbacks
+#else
+#define OVERHEAD_TICKS 185 // Non-32b data widths take longer to process
```

Comment:
these numbers are the same

Reply:
yes. that's confusing. going to leave it in for now as I suspect this test is a bit broken anyway, but it might just be that e.g. 16b is a pretty good case and that worst case is at like 21b or something.